### DISCUSSION TITLE
pygfs?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [NOAA-EMC/global-workflow#2925] [Q&A] [answered]
I am getting the following error:

```
+ JGLOBAL_STAGE_IC[7]: /groups/BCASH/global-workflow/scripts/exglobal_stage_ic.py
Traceback (most recent call last):
  File "/groups/BCASH/global-workflow/scripts/exglobal_stage_ic.py", line 5, in <module>
    from pygfs.task.stage_ic import Stage
ModuleNotFoundError: No module named 'pygfs'
```

I'm not turning up pygfs via either pip or conda-forge; how do I get this module?

**Comment by WalterKolczynski-NOAA** [comment, marked as answer]
It's already in workflow [ush/python/pygfs](https://github.com/NOAA-EMC/global-workflow/tree/develop/ush/python/pygfs) and should have already been included in the python path.

**Reply by benjamin-cash** [reply]
Thanks @WalterKolczynski-NOAA - I must have done something weird to my pythonpath when setting this up in the container.
